spec/spec_helper.rb

```diff
@@ -1,14 +1,9 @@
-require 'inherited_class_var'
-require 'bundler/setup'
-require 'coveralls'
-
-begin
-  require 'pry'
-rescue LoadError
-end
-
+Bundler.require(:default, :test)
 Coveralls.wear!
 
+$LOAD_PATH.unshift File.expand_path('../../lib', __FILE__)
```

Comment:
lol, we talked about this before---i prefer it, but it seems like it bugs ya. i'll remove it just for you ;)